awscli/examples/elasticbeanstalk/update-environment.rst

```diff
@@ -35,6 +35,9 @@ The following command sets the value of the "PARAM1" variable in the "my-env" en
 
 The ``option-settings`` parameter takes a namespace in addition to the name and value of the variable. Elastic Beanstalk supports several namespaces for options in addition to environment variables.
 
+**To set multiple environment variables in shorthand form**
```

Comment:
Nit: Can we change this to `**To set multiple environment variables using shorthand syntax**`? That way, we are consistent with the vocabulary we use when referring to "shorthand syntax".